src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -55,8 +55,8 @@ public StorageConfiguration()
         [Description("Time to sync after flash in seconds")]
         [DefaultValue(30)]
         [TimeUnit(TimeUnit.Seconds)]
-        [ConfigurationEntry("Storage.TimeToSyncAfterFlashInSec", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         [ConfigurationEntry("Storage.TimeToSyncAfterFlushInSec", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+        [ConfigurationEntry("Storage.TimeToSyncAfterFlashInSec", ConfigurationEntryScope.ServerWideOrPerDatabase)]
```

Comment:
what the purpose of having same name twice here?

Reply:
yaa - it is like 'where's wally' :))
one is with 'Flush' and one is with 'Flash'

Reply:
lol, was checking that 3 times and didn't found the difference. You are absolutely right. :)